planetiler-core/src/test/java/com/onthegomap/planetiler/util/HashingTest.java

```diff
@@ -0,0 +1,46 @@
+package com.onthegomap.planetiler.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+class HashingTest {
+
+  @ParameterizedTest
+  @ArgumentsSource(TestArgs.class)
+  void testFnv32(boolean expectSame, byte[] data0, byte[] data1) {
+    var hash0 = Hashing.fnv32(data0);
```

Comment:
hmm, I am a big fan of parameterized tests and the argument provider could have been re-used for other hash function but I rewrote it now - tho it's not so compact because of the required cast.